src/plat-net/posix/nyoci-plat-net.c

```diff
@@ -815,6 +880,25 @@ nyoci_plat_wait(
 
 	descriptors_ready = poll(polls, poll_count, cms);
 
+#else // !NYOCI_CAN_POLL
+	fd_set reads, writes, errors;
+	int fdCount = 0;
+	FD_ZERO(&reads);
+	FD_ZERO(&writes);
+	FD_ZERO(&errors);
+	if (cms <= 0) {
+		cms = 3600 * MSEC_PER_SEC;
+	}
+	ret = nyoci_plat_update_fdsets(self, &reads, &writes, &errors, &fdCount, &cms);
+	if (ret != NYOCI_STATUS_OK)
+		goto bail;
```

Comment:
I know there is a lot of old code in LibNyoci that doesn't use braces for single-line `if()` statements, but ever since the [goto fail](https://en.wikipedia.org/wiki/Unreachable_code#goto_fail_bug) bug I require the use of curly braces for all `if()` statements in all *new* code, even if copy-pasted.